src/PrestaShopBundle/Bridge/Listing/FiltersHelper.php

```diff
@@ -26,19 +26,20 @@
 
 declare(strict_types=1);
 
-namespace PrestaShopBundle\Bridge\Helper;
+namespace PrestaShopBundle\Bridge\Listing;
 
 use Context;
 use ObjectModel;
 use PrestaShop\PrestaShop\Adapter\LegacyContext;
 use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
-use PrestaShopBundle\Bridge\AdminController\FilterPrefix;
 use Symfony\Component\HttpFoundation\Request;
 use Tools;
 use Validate;
 
 /**
  * This class processes filters, stores them in cookies and updates the list's SQL query.
+ *
+ * @todo: rename to handler/processor?
```

Comment:
`FiltersProcessor` seems fine and the method should be renamed `processFilters` with an S